apps/prairielearn/src/migrations/20241007203121_draft_questions.sql

```diff
@@ -0,0 +1,34 @@
+ALTER TABLE questions
+ADD COLUMN IF NOT EXISTS is_draft boolean NOT NULL DEFAULT (false);
+
+ALTER TABLE pl_courses
+ADD COLUMN IF NOT EXISTS draft_number integer NOT NULL DEFAULT (0);
+
+CREATE TABLE IF NOT EXISTS questions_draft_metadata (
```

Comment:
@mwest1066 this table name feels off to me somehow. Is it the leading plural? The fact that it's `questions_draft` instead of `draft_question`? Something else? Let's bikeshed on this a bit.

Reply:
changed in 562ccc7ca4238b061cc5fed79757979f8c0db35b.